diff -up ParaView-3.14.0-RC2/Utilities/Xdmf2/vtk/Testing/Cxx/XdmfTestVTKIO.cxx.gcc47 ParaView-3.14.0-RC2/Utilities/Xdmf2/vtk/Testing/Cxx/XdmfTestVTKIO.cxx
--- ParaView-3.14.0-RC2/Utilities/Xdmf2/vtk/Testing/Cxx/XdmfTestVTKIO.cxx.gcc47	2012-02-09 10:43:55.000000000 -0700
+++ ParaView-3.14.0-RC2/Utilities/Xdmf2/vtk/Testing/Cxx/XdmfTestVTKIO.cxx	2012-02-11 14:30:02.946878437 -0700
@@ -37,6 +37,8 @@
 #include "vtkTimeSourceExample.h"
 #include "vtkXdmfReader.h"
 
+#include <unistd.h>
+
 #define NUMTESTS 20
 
 const char testobject[NUMTESTS][40] = {
diff -up ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeTools.cxx.gcc47 ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeTools.cxx
--- ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeTools.cxx.gcc47	2012-02-09 10:45:19.000000000 -0700
+++ ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeTools.cxx	2012-02-11 13:53:22.515115770 -0700
@@ -31,6 +31,8 @@
 // The embedded fonts
 #include "fonts/vtkEmbeddedFonts.h"
 
+#include <unistd.h>
+
 #ifdef FTGL_USE_NAMESPACE
 using namespace ftgl;
 #endif
diff -up ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeUtilities.cxx.gcc47 ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeUtilities.cxx
--- ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeUtilities.cxx.gcc47	2012-02-09 10:45:19.000000000 -0700
+++ ParaView-3.14.0-RC2/VTK/Rendering/vtkFreeTypeUtilities.cxx	2012-02-11 11:34:44.396589034 -0700
@@ -31,6 +31,7 @@
 #include "fonts/vtkEmbeddedFonts.h"
 
 #include <sys/stat.h>
+#include <unistd.h>
 
 #ifdef FTGL_USE_NAMESPACE
 using namespace ftgl;
