diff -up ParaView/ParaViewCore/ServerImplementation/Core/CMakeLists.txt.Protobuf ParaView/ParaViewCore/ServerImplementation/Core/CMakeLists.txt
--- ParaView/ParaViewCore/ServerImplementation/Core/CMakeLists.txt.Protobuf	2012-11-27 09:09:41.000000000 -0700
+++ ParaView/ParaViewCore/ServerImplementation/Core/CMakeLists.txt	2012-11-27 13:00:36.072178577 -0700
@@ -80,16 +80,16 @@ set (${vtk-module}_HDRS
 #------------------------------------------------------------------------------
 # Generate the protbuf message file.
 #------------------------------------------------------------------------------
-get_target_property(PROTOC_LOCATION protoc_compiler LOCATION)
+get_target_property(PROTOC_LOCATION protoc LOCATION)
 add_custom_command(
     OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/vtkPVMessage.pb.h
            ${CMAKE_CURRENT_BINARY_DIR}/vtkPVMessage.pb.cc
     
-    COMMAND protoc_compiler
+    COMMAND protoc
            "--cpp_out=dllexport_decl=VTKPVSERVERIMPLEMENTATIONCORE_EXPORT:${CMAKE_CURRENT_BINARY_DIR}"
            --proto_path "@CMAKE_CURRENT_SOURCE_DIR@"
            "${CMAKE_CURRENT_SOURCE_DIR}/vtkPVMessage.proto"
-    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/vtkPVMessage.proto protoc_compiler
+    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/vtkPVMessage.proto
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 )
 
@@ -108,5 +108,7 @@ endif()
 vtk_module_library(vtkPVServerImplementationCore
   ${Module_SRCS})
 
+target_link_libraries(vtkPVServerImplementationCore -lpthread)
+ 
 add_dependencies(vtkPVServerImplementationCore
   protobuf_code_generation)
diff -up ParaView/ThirdParty/protobuf/CMakeLists.txt.Protobuf ParaView/ThirdParty/protobuf/CMakeLists.txt
--- ParaView/ThirdParty/protobuf/CMakeLists.txt.Protobuf	2012-11-27 09:09:45.000000000 -0700
+++ ParaView/ThirdParty/protobuf/CMakeLists.txt	2012-11-27 13:02:04.535632175 -0700
@@ -34,7 +34,7 @@ set (PROTOBUF_INSTALL_BIN_DIR ${VTK_INST
 set (PROTOBUF_INSTALL_LIB_DIR ${VTK_INSTALL_LIBRARY_DIR})
 set (PROTOBUF_INSTALL_EXPORT_NAME ${VTK_INSTALL_EXPORT_NAME})
 
-vtk_module_third_party(protobuf
+vtk_module_third_party(Protobuf
   INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/vtkprotobuf/src
   LIBRARIES protobuf
 )
@@ -42,11 +42,11 @@ vtk_module_third_party(protobuf
 # protobuf exports it's build-dir targets to a custom file
 # (PROTOBUF_EXPORTS.cmake). We don't care much about that. We export
 # build-dir targets ourselves.
-vtk_target_export(protobuf)
-vtk_target_export(protobuf-lite)
-if (NOT CMAKE_CROSSCOMPILING)
-  vtk_compile_tools_target_export(protoc_compiler)
-endif()
+#vtk_target_export(protobuf)
+#vtk_target_export(protobuf-lite)
+#if (NOT CMAKE_CROSSCOMPILING)
+#  vtk_compile_tools_target_export(protoc_compiler)
+#endif()
 
 # All these exports don't add any install rules. However we make protobuf itself
 # install components at the right location by setting PROTOBUF_INSTALL_*
