--- ParaView-3.14.1-Source/VTK/Infovis/vtkBoostBreadthFirstSearchTree.cxx	2012-04-02 10:57:30.000000000 -0600
+++ VTK5.10.1/Infovis/vtkBoostBreadthFirstSearchTree.cxx	2012-10-15 14:33:21.000000000 -0600
@@ -49,11 +49,17 @@
 
 #if BOOST_VERSION >= 104800      // Boost 1.48.x
 namespace {
-  vtkIdType unwrap_edge_id(vtkEdgeType const &e) {
+  vtkIdType unwrap_edge_id(vtkEdgeType const &e)
+  {
     return e.Id;
   }
-  vtkIdType unwrap_edge_id(boost::detail::reverse_graph_edge_descriptor<vtkEdgeType> const &e) {
+  vtkIdType unwrap_edge_id(boost::detail::reverse_graph_edge_descriptor<vtkEdgeType> const &e)
+  {
+# if BOOST_VERSION == 104800
     return e.underlying_desc.Id;
+# else
+    return e.underlying_descx.Id;
+# endif
   }
 }
 #endif
