diff -up VTK/CMakeLists.txt.system VTK/CMakeLists.txt
--- VTK/CMakeLists.txt.system	2012-12-03 08:31:29.000000000 -0700
+++ VTK/CMakeLists.txt	2012-12-06 09:58:22.809749444 -0700
@@ -97,6 +97,11 @@ set(VTK_INSTALL_INCLUDE_DIR_CM24 ${VTK_I
 set(VTK_INSTALL_DOXYGEN_DIR_CM24 ${VTK_INSTALL_DATA_DIR}/doxygen)
 
 #-----------------------------------------------------------------------------
+# Do we try to use system libraries by default?
+OPTION(VTK_USE_SYSTEM_LIBRARIES "Use the system's libraries by default." OFF)
+MARK_AS_ADVANCED(VTK_USE_SYSTEM_LIBRARIES)
+
+#-----------------------------------------------------------------------------
 # The third party macros are still used in one or two third party builds.
 include(vtkThirdParty)
 
diff -up VTK/CMake/vtkModuleMacros.cmake.system VTK/CMake/vtkModuleMacros.cmake
--- VTK/CMake/vtkModuleMacros.cmake.system	2012-12-03 08:31:29.000000000 -0700
+++ VTK/CMake/vtkModuleMacros.cmake	2012-12-06 09:58:22.810749439 -0700
@@ -563,7 +563,7 @@ macro(vtk_module_third_party _pkg)
     message(FATAL_ERROR "Cannot specify both LIBRARIES and NO_LIBRARIES")
   endif()
 
-  option(VTK_USE_SYSTEM_${_upper} "Use system-installed ${_pkg}" OFF)
+  option(VTK_USE_SYSTEM_${_upper} "Use system-installed ${_pkg}" ${VTK_USE_SYSTEM_LIBRARIES})
   mark_as_advanced(VTK_USE_SYSTEM_${_upper})
 
   if(VTK_USE_SYSTEM_${_upper})
