diff -up ParaView-3.12.0/VTK/Infovis/vtkBoostBreadthFirstSearchTree.cxx\~ ParaView-3.12.0/VTK/Infovis/vtkBoostBreadthFirstSearchTree.cxx
--- ParaView-3.12.0/VTK/Infovis/vtkBoostBreadthFirstSearchTree.cxx~	2011-11-08 21:56:37.000000000 +0100
+++ ParaView-3.12.0/VTK/Infovis/vtkBoostBreadthFirstSearchTree.cxx	2011-12-06 01:11:42.487760688 +0100
@@ -47,6 +47,15 @@ using namespace boost;
 vtkStandardNewMacro(vtkBoostBreadthFirstSearchTree);
 
 
+namespace {
+  vtkIdType unwrap_edge_id(vtkEdgeType const &e) {
+    return e.Id;
+  }
+  vtkIdType unwrap_edge_id(boost::detail::reverse_graph_edge_descriptor<vtkEdgeType> const &e) {
+    return e.underlying_desc.Id;
+  }
+}
+
 // Redefine the bfs visitor, the only visitor we
 // are using is the tree_edge visitor.
 template <typename IdMap>
@@ -95,7 +104,8 @@ public:
 
     // Copy the vertex and edge data from the graph to the tree.
     tree->GetVertexData()->CopyData(graph->GetVertexData(), v, tree_v);
-    tree->GetEdgeData()->CopyData(graph->GetEdgeData(), e.Id, tree_e.Id);
+    tree->GetEdgeData()->CopyData(graph->GetEdgeData(),
+				  unwrap_edge_id(e), tree_e.Id);
   }
 
 private:

Diff finished.  Tue Dec  6 01:13:13 2011
